Description: Explain to the user how to fix the boinc startup issue
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Last-Update: 2020-09-18

Index: boinc/lib/gui_rpc_client.cpp
===================================================================
--- boinc.orig/lib/gui_rpc_client.cpp
+++ boinc/lib/gui_rpc_client.cpp
@@ -422,7 +422,7 @@
         char path[MAXPATHLEN];
         if (errno == EACCES) {
             sprintf(msg_buf,
-                "%s exists but can't be read.  Check the file permissions.",
+                "%s exists but can't be read.  Check the file permissions, or ask the system administrator to add the user in the 'boinc' group.",
                 GUI_RPC_PASSWD_FILE
             );
             msg = msg_buf;
@@ -448,7 +448,7 @@ int read_gui_rpc_password(char* buf, str
                 if (!f) {
                     if (errno == EACCES) {
                         sprintf(msg_buf,
-                            "%s exists but can't be read.  Check the file permissions.",
+                            "%s exists but can't be read.  Check the file permissions, or ask the system administrator to add the user in the 'boinc' group.",
                             path
                         );
                     } else {
@@ -475,7 +475,7 @@ int read_gui_rpc_password(char* buf, str
             if (!f) {
                 if (errno == EACCES) {
                     sprintf(msg_buf,
-                        "%s exists but can't be read.  Check the file permissions.",
+                        "%s exists but can't be read.  Check the file permissions, or ask the system administrator to add the user in the 'boinc' group.",
                         path
                     );
                     msg = msg_buf;
